core/src/main/java/bisq/core/btc/model/RawTransactionInput.java

```diff
@@ -27,12 +27,16 @@
 
 import javax.annotation.concurrent.Immutable;
 
+/**
+ * Holds the relevant data for the connected output for a tx input.
+ * ParentTransaction means here the spending tx, the tx of the output not the parentTransaction of the input itself.
+ * Also index refers to the index of the spending output
+ */
```

Comment:
I would reformat the method javadoc with some minor grammar changes:
```
    /**
     * Holds the relevant data for the connected output for a tx input.
     * @param index  the index of the parentTransaction
     * @param parentTransaction  the spending output tx, not the parent tx of the input
     * @param value  the number of satoshis being spent
     */

```

Reply:
But `ACK`, it's only a comment.

Reply:
Thanks, will do a commit a bit later (in the middle of some other works atm...)